pkg/apis/application/v1alpha1/repository_types.go

```diff
@@ -0,0 +1,290 @@
+package v1alpha1
+
+import (
+	"net/url"
+	"regexp"
+
```

Comment:
```
Check failure on line 6 in pkg/apis/application/v1alpha1/repository_types.go

GitHub Actions
/ Lint Go code
pkg/apis/application/v1alpha1/repository_types.go#L6
File is not `goimports`-ed with -local github.com/argoproj/argo-cd (goimports)
```

No clue what this means :(

Reply:
It means that imports are not ordered as the linter would have expected. The linter assumes the imports come in three separate blocks, each of which is alphabetically sorted:

* Block 1: Golang standard library imports (e.g. `net/http`)
* Block 2: Imports from same module (e.g. `github.com/argoproj/argo-cd/util/cert`)
* Block 3: All other external package imports (e.g. `log "github.com/sirupsen/logrus"`)

All blocks should be separated by one blank line. So, in your case, there's just a blank line missing between the following two lines:

```
        "github.com/argoproj/argo-cd/util/helm"
	log "github.com/sirupsen/logrus"
```

:)

Reply:
😱 Thank you so much!